ykman/cli/openpgp.py

```diff
@@ -183,6 +183,60 @@ def set_pin_retries(
             echo_default_pins()
 
 
+@access.command("change-pin")
+@click.option("-P", "--pin", help="Current PIN code.")
+@click.option("-n", "--new-pin", help="A new PIN.")
+@click.pass_context
+def change_pin(ctx, pin, new_pin):
+    """
+    Change PIN.
+
+    The PIN must be atleast 6 long, and supports any type of
```

Comment:
"at least" should be written as two words. "6 long" needs a unit. I would suggest "at least 6 characters long", but that repeats the word "characters" in the sentence. Maybe "The PIN has a minimum length of 6, and supports any type of alphanumeric characters."?